static/app/components/events/contexts/app/index.tsx

```diff
@@ -26,27 +30,34 @@ export const appKnownDataValues = [
   AppKnownDataType.IN_FOREGROUND,
 ];
 
-const appIgnoredDataValues = [];
+const appIgnoredDataValues: string[] = [];
```

Comment:
It doesn't look like you added this but I'm confused why it's here 😕

Reply:
When I was iterating on this PR, I tested a bunch of ways of passing around the functions with `AppContext` as the test space since it's first alphabetically. For one of them I had to type that array since it was cast as `never[]`, but now it's just outdated. Will remove, great catch!